packages/monorepo-scripts/src/publish.ts

```diff
@@ -38,8 +38,14 @@ const packageNameToWebsitePath: { [name: string]: string } = {
 };
 
 (async () => {
+    const hasRequiredSetup = await checkPublishRequiredSetupAsync();
+    if (!hasRequiredSetup) {
+        return; // abort
```

Comment:
Wouldn't it be better to throw a helpfull error message then to just silently fail?

Reply:
We print helpful messages for each conditions within `checkPublishRequiredSetupAsync`. Do you still think this is necessary?